src/Mod/Start/Gui/StartView.cpp

```diff
@@ -260,6 +248,17 @@ void StartView::configureNewFileButtons(QLayout* layout) const
     connect(arch, &QPushButton::clicked, this, &StartView::newArchFile);
 }
 
+void StartView::paintEvent(QPaintEvent* event)

```

Comment:
If this seems silly, it is just the same fix as was already used in `FileCardDelegate.cpp`:

https://github.com/FreeCAD/FreeCAD/blob/ef2156d0f6d116b1642a5531a2d44fc6b7d17042/src/Mod/Start/Gui/FileCardDelegate.cpp#L85-L88

https://github.com/FreeCAD/FreeCAD/blob/ef2156d0f6d116b1642a5531a2d44fc6b7d17042/src/Mod/Start/Gui/FileCardDelegate.cpp#L115-L129